website/static/js/search.js

```diff
@@ -116,12 +116,21 @@ var ViewModel = function(params) {
     });
 
     self.queryObject = ko.pureComputed(function(){
+        var TITLE_BOOST = '2';
+        var DESCRIPTION_BOOST = '1.2';
+
+        var fields = [
+            '_all',
+            'title^' + TITLE_BOOST,
+            'description^' + DESCRIPTION_BOOST,
+        ];
         return {
             'query_string': {
                 'default_field': '_all',
+                'fields': fields,
                 'query': self.query(),
                 'analyze_wildcard': true,
-                'lenient': true
+                'lenient': true,
```

Comment:
Trailing commas aren't valid in JS (even though they are encouraged in Python). 

Will make this change when I merge.